modules/op-release-notes-1-8.adoc

```diff
@@ -50,8 +238,141 @@ Workaround: Use `nodejs:14-ubi8` instead of the `nodejs:14-ubi8-minimal` image s
+----
++
+This update fixes the issue so that the failure no longer occurs.
+// (borrowed from 1.7.2 release notes) https://issues.redhat.com/browse/SRVKP-2327
+// Khurram Baig
+
+* Previously, upgrading the {pipelines-title} Operator caused `Pipeline`  objects to fail. This update fixes this issue. During upgrades, the Operator no longer recreates the `pipeline` service account. As a result, secrets attached to the `pipeline` service account persist over upgrades and the `Pipeline` objects continue working correctly.
+// https://issues.redhat.com/browse/SRVKP-2256
+// Kushagra Kulshreshtha
+
+* With this update, {pac} pods run on infrastructure nodes if infrastructure node settings are configured in the `TektonConfig` custom resource (CR).
+// no link provided
+
+* Previously, with the resource pruner, each namespace Operator created a command that ran in a separate container. This design consumed too many resources in clusters with a high number of namespaces. For example, to run a single command, a cluster with 1000 namespaces produced 1000 containers in a pod.
++
+This update fixes the issue. It passes the namespace-based configuration to the job so that all the commands run in one container in a loop. (link:https://issues.redhat.com/browse/SRVKP-2160[SRVKP-2160])
+// Pradeep Kumar
+
+* In {tekton-chains} you must define a secret called `signing-secrets` to hold the key used for signing tasks and images. However, before this update, updating the {pipelines-title} Operator reset or overwrote this secret and the key was lost. This update fixes the issue. Now, if the secret is configured after installing {tekton-chains} through the Operator, the secret persists, and it is not overwritten by upgrades.
+// https://issues.redhat.com/browse/SRVKP-2304
+// Piyush Garg
+
+* Before this update, all S2I build tasks failed with an error similar to the following message:
++
+----
+Error: error writing "0 0 4294967295\n" to /proc/22/uid_map: write /proc/22/uid_map: operation not permitted
+time="2022-03-04T09:47:57Z" level=error msg="error writing \"0 0 4294967295\\n\" to /proc/22/uid_map: write /proc/22/uid_map: operation not permitted"
+time="2022-03-04T09:47:57Z" level=error msg="(unable to determine exit status)"
+----
++
+With this update, the `pipelines-scc` security context constraint (SCC) is compatible with the `SETFCAP` capability necessary for S2I and Buildah cluster tasks. As a result, the S2I build tasks run successfully.
++
+// (borrowed from 1.7.2 release notes) https://issues.redhat.com/browse/SRVKP-2091
+// Piyush Garg
+
+* Before this update, installing the {product title} Operator took longer than expected. This update optimizes some settings to make installing the Operator go faster.
+// https://issues.redhat.com/browse/SRVKP-2369
```

Comment:
@sm43 Please review and approve. Thanks.

Reply:
looks good